torch/_tensor_docs.py

```diff
@@ -1746,12 +1747,14 @@ def callable(a, b) -> number
 reshape(*shape) -> Tensor
 
 Returns a tensor with the same data and number of elements as :attr:`self`,
-but with the specified shape.
+but with the specified shape. This method returns a view if :attr:`shape` is
+compatible with the current shape.
```

Comment:
"compatible" == same shape? If so we should clarify more

Reply:
I will add a pointer to the `view` doc. It is explained more over there.

Reply:
(y)